src/Widgets/ThumbnailGalleryWidget/ThumbnailGalleryWidgetComponent.js

```diff
@@ -124,19 +179,36 @@ function allTags(images) {
   return uniqueTags.sort();
 }
 
-function lightboxOptions(galleryImageWidget) {
-  const image = galleryImageWidget.get("image");
-  const binary = image.get("blob");
-  const srcUrl = binary.optimizeFor({ width: fullScreenWidthPixels() }).url();
-  const alt = image.get("alternativeText");
-
-  return {
-    photo: srcUrl,
-    thumbnail: srcUrl,
-    caption: [
-      galleryImageWidget.get("title"),
-      galleryImageWidget.get("subtitle"),
-    ].join(" - "),
-    alt,
-  };
+function SliderNextArrow(props) {
```

Comment:
* [ ] FYI Other components in this file use parameter destructuring. For consistency we could say `{ onClick }` instead of `props`.